openpype/plugins/publish/integrate.py

```diff
@@ -761,8 +759,8 @@ def prepare_representation(self, repre,
                     template_data["udim"] = index
                 else:
                     template_data["frame"] = index
-                anatomy_filled = anatomy.format(template_data)
-                template_filled = anatomy_filled[template_name]["path"]
+                template_obj = anatomy.templates_obj[template_name]["path"]
```

Comment:
This could be moved before the huge condition, so the variable `template_obj` is defined only once?

Reply:
Maybe even split it into 2 variables?
```
path_template = anatomy.templates_obj[template_name]["path"]
folder_template = anatomy.templates_obj[template_name]["folder"]
```